### DISCUSSION TITLE
Migration AWAY from Migration Guide?

### DISCUSSION THREAD

**Discussion by sarah11918** [withastro/docs#586] [Ideas]
Migration will be less of an issue as we get closer to and past v1, and I think it will be sufficient to refer to "updating!" or "What's new!" 

I'd like to put on our radar pruning (eventually removing?) the Migration docs, and/or giving them less prominence going forward. We have some migrations TO things in early versions that we have since yet migrated away FROM. So, this can be a potential source of confusion for people arriving via search results (either external, or within our own Algolia search.)

I also think What's New? is more attention getting, even for people on relatively new versions of Astro, who might be compelled to look there "just to see" vs clicking on a Migration Guide, which sounds more formal and like something you'd read if you had a purpose.

So, just putting this on our radar!

**Comment by delucis** [comment]
Relatedly, I noticed our Getting Started page currently links to the migration guide basically before anything else. That seems like something we can stop doing long before we can remove the guide itself. Maybe even already?

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

Well, 2022 was a long time ago... and I'm not sure what this discussion refers to since I wasn't there yet... 😅  I think it's safe to close!

If you think this is a mistake and that this discussion should remain open, please feel free to reopen!
